### PR TITLE
Bugfix lang

### PR DESCRIPTION
**Pull request by taspinar**

### CODE REVIEW

**Comment by sils** [comment]
this breaks semver since the API of `query_tweets` is changed, an upstream dependency would break when using the compatability operator `~=` and the api.

File: twitterscraper/main.py [hunk 1]
```diff
@@ -82,13 +82,10 @@ def main():
...
             exit(-1)
         
-        if args.lang:
-            args.query = "{}&l={}".format(args.query, args.lang)
-        
         if args.all:
             args.begindate = dt.date(2006,3,1)
 
-        tweets = query_tweets(args.query, args.limit, args.begindate, args.enddate, args.poolsize)
+        tweets = query_tweets(args.query, args.limit,args.lang, args.begindate, args.enddate, args.poolsize)
```
Comment (sils) [inline twitterscraper/main.py]: I highly recommend to use keyword arguments here, this breaks easy and is nonexplicit (see also zen of python)

File: twitterscraper/query.py [hunk 2]
```diff
@@ -87,8 +87,8 @@ def query_tweets_once(query, limit=None):
     try:
         while True:
             new_tweets, pos = query_single_page(
-                INIT_URL.format(q=query) if pos is None
-                else RELOAD_URL.format(q=query, pos=pos),
+                INIT_URL.format(q=query, lang=lang) if pos is None
```
Comment (sils) [inline twitterscraper/query.py]: architecture idea, take `**kwargs` and pass them like that to format, that'll always allow the user to pass everything that you allow in the URL format string.

**Comment by sils** [comment]
I definitely advise against releasing this. If this was already released, move the lang argument to the end and do another bugfix release to make the API backwards compatible at least (0.4.2, not 0.5 because it needs to be fixed for downstream users!)

File: twitterscraper/main.py (hunk 1 above)
Comment (taspinar) [reply twitterscraper/main.py]: Would it become something like ```query_tweets(args.query, ***vars(args))``` ?

Comment (taspinar) [reply twitterscraper/main.py]: See commit https://github.com/taspinar/twitterscraper/pull/73/commits/c96b7423e9858feac729f8fa9bbab442fdca77df

File: HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,14 @@
 ## 0.x.x
 
 TBD
+## 0.4.2 ( 2018-01-9 )
```
Comment (sils) [inline HISTORY.rst]: 9 -> 09

File: twitterscraper/query.py [hunk 4]
```diff
@@ -135,7 +135,7 @@ class NoElement: pass
 def roundup(numerator, denominator):
     return numerator // denominator + (numerator % denominator > 0)
 
-def query_tweets(query, limit = None, begindate = dt.date(2017,1,1), enddate = dt.date.today(), poolsize = 20):
+def query_tweets(query, limit = None, begindate = dt.date(2017,1,1), enddate = dt.date.today(), poolsize = 20, lang = ''):
```
Comment (sils) [inline twitterscraper/query.py]: you're not using spacing around `=` consistently, PEP8 suggests no spaces around `=` within param lists

**Comment by sils** [comment]
technically good otherwise IMO :+1:

**Comment by sils** [comment]
(commits are a bit chaotic and messages don't explain why, consider squashing when merging or `rebase -i` to clean up)

**Comment by sils** [comment]
(with messages I mean specifically shortlog)

**Pull request merged**